### PR TITLE
Update to FS2 0.10.0-M9

### PR DESCRIPTION
**Pull request by guersam**
- Update FS2 to 0.10.0-M9
- Drop `doobie.syntax.MonadErrorOps#onError` in favor of
  `cats.syntax.ApplicativeErrorOps#onError`

### CODE REVIEW

**Comment by tpolecat** [comment]
PR looks good. Not sure what's up with Travis. Ruby something.

**Comment by guersam** [comment]
I see. Why does the travis config require jekyll BTW? Is it a pre-tut stuff for the old docs?

**Comment by guersam** [comment]
I guessed wrong, it was added recently for microsite... Bumped ruby version to meet the failing gem constraint for now.

**Comment by guersam** [comment]
I'm not sure of where to start for the failed test :(

**Comment by tpolecat** [comment]
I won't have time to look at this for a few days.

File: modules/core/src/main/scala/doobie/util/transactor.scala [hunk 1]
```diff
@@ -50,12 +52,16 @@ object transactor  {
 
     /** Natural transformation that wraps a `ConnectionIO` program. */
     val wrap = λ[ConnectionIO ~> ConnectionIO] { ma =>
-      (before *> ma <* after) onError oops guarantee always
+      (before *> ma <* after)
+        .onError { case NonFatal(_) => oops }
+        .guarantee(always)
```
Comment (guersam) [inline modules/core/src/main/scala/doobie/util/transactor.scala]: My bad, it was a simple mistake in the original PR:
```scala
(before *> ma <* after) onError { 
  case NonFatal(_) => oops guarantee always 
}
```

Comment (tpolecat) [reply modules/core/src/main/scala/doobie/util/transactor.scala]: 👍

**Comment by tpolecat** [comment]
Thanks!

**Pull request merged**